packages/checkbox/test/benchmark/test-basic.ts

```diff
@@ -15,5 +15,7 @@ import { html } from 'lit';
 import { measureFixtureCreation } from '../../../../test/benchmark/helpers.js';
 
 measureFixtureCreation(html`
-    <sp-checkbox id="checkbox0" tabindex="5">Component</sp-checkbox>
+    <sp-checkbox id="checkbox1">Component</sp-checkbox>
+    <sp-checkbox id="checkbox2" checked>Component</sp-checkbox>
+    <sp-checkbox id="checkbox3" indeterminate>Component</sp-checkbox>
```

Comment:
Should this change be merged first in order to have a baseline that matches? If the benchmark has been changed in this branch only, won't the results be off? Perhaps it's even faster than the current reported results!

Reply:
Baseline is run against the previously release version, every time.

Reply:
We collapse them by default, but you can find the most recent run in [this comment](https://github.com/adobe/spectrum-web-components/pull/3775#issuecomment-1795008308).

Reply:
I see, so it's running this `test-basic.ts` against release `<sp-checkbox>`, then running it against the branch `<sp-checkbox>`, so there's no need update benchmarks in main for a fair comparison. Got it!